ui/src/app/state/user/user.selectors.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { User } from "src/app/services";
+import { UserState } from "./user.reducer";
+import { selectUser } from "./user.selectors";
+
+interface AppState {
+  user: UserState
+}
+
+describe('User Selectors', () => {
+  it('should return the user from the state', () => {
+    const user: User = {
+      name: 'Max'
+    };
+    const appState: AppState = {
+      user: {
```

Comment:
looks bit weird.
we could adjust userstate to be just extending props of the user dto?